fusor-ember-cli/app/routes/deployment.js

```diff
@@ -69,8 +69,7 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
             },
             success(response) {
               resolve(response);
-              var uuid = response.id;
-              deployment.set('foreman_task_uuid', uuid);
+              deployment.set('foreman_task', response);
```

Comment:
@jwmatthews, I added another commit that sets the belongsTo relationship `foreman_task` rather than the attribute that is no longer `foreman_task_uuid`. Does this work?